GrocifyApp.UI/src/pages/Products/ProductSections.jsx

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect } from 'react';
+import { ReactSVG } from 'react-svg';
+import { useNavigate } from 'react-router-dom';
+
+//Internal components
+import CustomInput from '../../components/CustomInput';
+import Layout from '../../components/Layout/Layout';
+
+//Assets & Css
+import SearchIcon from '../../assets/search-ic.svg';
+import DotsIcon from '../../assets/3-dots-ic.svg';
+import ChevronIcon from '../../assets/chevron-ic.svg';
+import PlusCircleIcon from '../../assets/plus-circle-ic.svg';
+
+//Consts
+import { PlaceholderConsts } from '../../consts/ENConsts';
+import { ButtonConsts } from '../../consts/ENConsts';
+import IconsConsts from "../../consts/IconsConsts";
+import ApiEndpoints from '../../consts/ApiEndpoints';
+import styles from './ProductSections.module.scss';
+
+function ProductSections() {
+    const [searchInput, setSearchInput] = useState('');
+    const [sections, setSections] = useState([]);
+    const token = localStorage.getItem('token');
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await fetch(ApiEndpoints.ProductSections_Endpoint, {
+                    method: 'GET',
+                    headers: {
+                        'Content-Type': 'application/json',
+                        'Authorization': `Bearer ${token}`,
+                    },
+                });
+
+                if (!response.ok) {
+                    throw new Error(`Failed to fetch data: ${response.statusText}`);
+                }
+                                
+                const data = await response.json();
+                setSections(data);
+            } catch (error) {
+                console.log('');
+            }
+        };
+
+        fetchData();
+    }, []);
```

Comment:
can you remove the ", []"?
because it is giving a warning:

![image](https://github.com/Catarina98/Grocify/assets/36730010/53ed0b80-89fd-47aa-8ce1-c99002616aea)